CryptoPrices/Sources/App/CryptoPricesApp.swift

```diff
@@ -6,15 +6,33 @@
 //
 
 import Home
+import MyCoin
 import Styleguide
 import SwiftUI
 
 @main
 struct CryptoPricesApp: App {
 
+    @StateObject var appCoordinator = AppCoordinator()
+    
+    @ViewBuilder var rootView: some View {
+        switch appCoordinator.state {
+        case let .home(homeState):
+            HomeView()
+                .environmentObject(homeState)
+        case let .myCoin(myCoinState):
+            MyCoinView()
+                .environmentObject(myCoinState)
+                .transition(
+                    .move(edge: .trailing)
+                    .animation(.linear(duration: 1.0))
+                )
+        }
```

Comment:
We are going to have a long list of screens here the more screens we have I believe 😅

Reply:
As I said ☝️, a long list of `switch` here when we have multiple screens/scenes in our `enum Scene` is naturally normal as it's the navigation code 🙏